cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/resource/cruisecontrol/PathBuilder.java

```diff
@@ -89,6 +117,8 @@ private void addIfNotZero(PathBuilder builder, CruiseControlParameters param, lo
     public PathBuilder withRebalanceParameters(RebalanceOptions options) {
         if (options != null) {
             PathBuilder builder = withAbstractRebalanceParameters(options)
+                    .withParameter(CruiseControlParameters.VERBOSE, String.valueOf(options.isVerbose()))
+                    .withParameter(CruiseControlParameters.SKIP_HARD_GOAL_CHECK, String.valueOf(options.isSkipHardGoalCheck()))
```

Comment:
what's the need for this change on a totally different endpoint not related to the remove-disks one?
Same comment for all the following ones, they work fine as they are why are we changing? Or did you find something we should do but tbh it should be in a separate PR?

Reply:
This change is introduced since I was trying to use `withAbstractRebalanceParameters` with all the endpoints like remove-disks, remove-broker and `add-broker` etc